doc/examples.xml

```diff
@@ -309,3 +309,36 @@ gap> GeneralisedPetersenGraph(IsMutableDigraph, 9, 4);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="TadpoleDigraph">
+<ManSection>
+  <Oper Name="TadpoleDigraph" Arg="[filt, ]m, n"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    If <A>m</A> and <A>n</A> are non-negative integers (where <A>m</A>> 2), then this operation
+    returns a Tadpole digraph.<P/>
+
+    If the optional first argument <A>filt</A> is present, then this should
+    specify the category or representation the digraph being created will
+    belong to. For example, if <A>filt</A> is <Ref Filt="IsMutableDigraph"/>,
+    then the digraph being created will be mutable, if <A>filt</A> is <Ref
+    Filt="IsImmutableDigraph"/>, then the digraph will be immutable. 
+    If the optional first argument <A>filt</A> is not present, then <Ref
+      Filt="IsImmutableDigraph"/> is used by default.<P/>
+
+    The <E>Tadpole digraph</E> consists of a cycle graph on <A>m</A> vertices and a path graph on
+    <A>n</A> vertices. These two graphs are connected by a bridge.<P/>
```

Comment:
Please put the description of the tadpole digraph before the previous paragraph (the one all about mutability).

Please can you also link to the Wikipedia entry https://en.wikipedia.org/wiki/Tadpole_graph (with `<URL>` tags).

You should also refer to the terms `cycle digraph` and `chain digraph` rather than `cycle graph` and `path graph`, since these are the standard terms we use in the Digraphs package. The term `bridge` is also undefined in this package, so perhaps you could avoid this terminology and just explicitly say what the bridge is.

Indeed, I think overall you should also be more explicit about which vertices and edges correspond to which part of the tadpole. So something along the lines of:

```
The <E>Tadpole digraph</E> is the symmetric closure of the disjoint union
of the cycle digraph on <C>[1..<A>m</A>]</C> (the 'head' of the tadpole)
and the chain digraph on <C>[<A>m</A>+1..<A>m</A>+<A>n</A>]</C>
(the 'tail' of the tadpole), along with the additional
edges <C>[1, <A>m</A>+1]</C> and <C>[1, <A>m</A>+1]</C>
which connect the 'head' and the 'tail'. <P/>

See <Ref Oper="DigraphSymmetricClosure"/>, <Ref Oper="DigraphDisjointUnion"/>,
<Ref Oper="CycleDigraph"/>, and <Ref Oper="ChainDigraph"/>.
```
So I suppose I am also suggesting that you move the bridge from connecting the vertices `m` and `n + m` to connecting the vertices `1` and `m + 1` - but this is unimportant (as long as whatever you choose is documented correctly 🙂).